### ISSUE TITLE
How to specify animated textures?

### ISSUE THREAD

**Issue by zeman-88** [PixarAnimationStudios/USD#750]
Is it possible to specify animated textures for USD, specificially so that they can be animated when a USDZ file is created?

**Comment by spiffmon** [comment]
Hi,
   Providing multiple timeSamples for the _file_ input of a UsdUVTexture node should produce an animated texture in usdview.  However, I do not think the Apple renderers support such animated textures yet, and certainly their usdz_converter program does not.

**Comment by AlexanderPetrovichev** [comment]
@zeman-88 Hello! Did you manage to compile this USD library for iOS?

**Comment by enquel** [comment]
Hi there. Bumping up. Is there a way to have animated texture (or at least an uv scrollable substitute) that would work with Apple's AR Quick Look?

**Comment by spiffmon** [comment]
Hi, this is is question that only Apple’s developer support can answer.

**Comment by enquel** [comment]
@spiffmon thanks, that's what I assumed, only wanted to double check. Wonderful format, BTW!

**Comment by useronym** [comment]
@enquel Did you figure out a way to have an animated texture in AR QuickLook? Seems like somebody here made a custom GIF to USDZ converter, but no source is shared: https://www.youtube.com/watch?v=jap4nkfXVWE

**Comment by enquel** [comment]
@osense Unfortunately my R&D budget for this task was small so I had to stop. Officially it is not supported, iOS 13.7 added audio support, so I guess we have to wait for animated texture. My guess is this guy feeds single frames to .usdz in a way edible by AR Quick Look. Have you tried contacting him?

**Comment by useronym** [comment]
@enque yeah, unfortunately it seems like even if this approach was made to work it would not be practical in terms of model file size

https://twitter.com/PaulHamilton8/status/1304166262189092865
